### PR TITLE
layers: Add maxFragmentOutputAttachments

### PR DESCRIPTION
**Pull request by spencer-lunarg**
We never were checking for `maxFragmentOutputAttachments`, but we do now

### CODE REVIEW

File: tests/unit/shader_interface.cpp [hunk 1]
```diff
@@ -1490,6 +1490,10 @@ TEST_F(NegativeShaderInterface, MultidimensionalArray64bit) {
     RETURN_IF_SKIP(Init());
     InitRenderTarget();
 
+    if (m_device->phy().limits_.maxFragmentOutputAttachments < 25) {
```
Comment (artem-lunarg) [inline tests/unit/shader_interface.cpp:1493]: Do we have a configuration we this does not skip?

**Review by artem-lunarg — APPROVED** [review]
LGTM. One question about test.

File: tests/unit/shader_interface.cpp (hunk 1 above)
Comment (spencer-lunarg) [reply tests/unit/shader_interface.cpp:1493]: not yet... i'll explain offline

Comment (spencer-lunarg) [reply tests/unit/shader_interface.cpp:1493]: (at our next weekly meeting)

**Pull request merged**